mainwindow.cpp

```diff
@@ -273,105 +371,46 @@ void MainWindow::on_pushButton_13_clicked()
     ui->label_5->setStyleSheet("color:rgb(255,180,241)");
 }
 
+void MainWindow::automate_callback() {

```

Comment:
This function replaced the original while loop and QApplication::processEvent() in on_pushButton_14_clicked()